ep006/README.md

```diff
@@ -0,0 +1,40 @@
+# [python type(x).__name__ vs x.__class__.__name__ (intermediate)](https://youtu.be/6rAIttnm3Fs)
+
+Very short one, I explain why I prefer type(x).__name__ vs x.__class__.__name__.  thanks to @alikus for the question!
+
+## Interactive examples
+
+### Python
+
+```python
+# python3.8
+Location()
+class C:
+    @property
+    def __class__(self):
+        print('return custom type')
```

Comment:
Added debug print for better visibility of code execution. Let me if you want this to be removed as well.